internal/handlers/common/sort.go

```diff
@@ -14,10 +14,173 @@
 
 package common
 
-import "github.com/FerretDB/FerretDB/internal/types"
+import (
+	"math"
+	"sort"
+	"strings"
+
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+)
+
+// sortType represents sort type for $sort aggregation.
+type sortType int
+
+const (
+	ascending sortType = iota
+	descending
+)
 
 // SortDocuments sorts given documents in place according to the given sorting conditions.
 func SortDocuments(docs []*types.Document, sort *types.Document) error {
-	// TODO
+	if sort.Len() == 0 {
+		return nil
+	}
+
+	if sort.Len() > 32 {
+		return lazyerrors.Errorf("maximum sort keys exceeded: %v", sort.Len())
+	}
+
+	sortFuncs := make([]sortFunc, len(sort.Keys()))
+	for i, sortKey := range sort.Keys() {
+		sortField, err := sort.Get(sortKey)
+		if err != nil {
+			return err
+		}
+		sortType, err := getSortType(sortField)
+		if err != nil {
+			return err
+		}
+
+		sortKey := sortKey
+		sortFuncs[i] = func(a, b *types.Document) bool {
```

Comment:
Let's call it lessFuncs or something (and add a comment) because sorting function that accepts two documents and returns a boolean raises questions